src/main/java/org/killbill/billing/overdue/api/OverdueCondition.java

```diff
@@ -35,4 +35,6 @@
     public PaymentResponse [] getResponseForLastFailedPaymentIn();
 
     public ControlTagType getControlTagType();
```

Comment:
I am wondering if we should rename `getControlTagType` -> `getInclusionControlTagType` to make it symmetrical-- also might break XML compatibility ? @pierre Any thoughts?